chronos/apps/http/httpagent.nim

```diff
@@ -0,0 +1,24 @@
+#
+#        Chronos HTTP/S client implementation
+#             (c) Copyright 2021-Present
+#         Status Research & Development GmbH
+#
+#              Licensed under either of
+#  Apache License, version 2.0, (LICENSE-APACHEv2)
+#              MIT license (LICENSE-MIT)
+import strutils
+
+const
+  ChronosName* = "nim-chronos"
+    ## Project name string
+  ChronosMajor* {.intdefine.}: int = 3
+    ## Major number of Chronos' version.
+  ChronosMinor* {.intdefine.}: int = 0
+    ## Minor number of Chronos' version.
+  ChronosPatch* {.intdefine.}: int = 2
```

Comment:
isn't there a way to share this with the version information in the nimble file? I seem to remember @zah had some magic tric